tests/cli/test_compose_sanity.sh

```diff
@@ -22,6 +22,27 @@ rlJournalStart
     rlPhaseEnd
 
     rlPhaseStartTest "compose start"
+        UUID=`$CLI compose start example-http-server ami`
+        rlAssertEquals "exit code should be zero" $? 0
+        UUID=`echo $UUID | cut -f 2 -d' '`
+
+        if [ -n "$UUID" ]; then
+            until $CLI compose info $UUID | grep 'RUNNING'; do
+                sleep 20
+                rlLogInfo "Waiting for compose to start ..."
+            done;
+        else
+            rlFail "Compose UUID is empty!"
+        fi
+    rlPhaseEnd
+
+    rlPhaseStartTest "cancel compose"
+        rlRun -t -c "$CLI compose cancel $UUID"
+        `$CLI compose status | grep $UUID`
+        rlAssertNotEquals "compose is canceled" $? 0
```

Comment:
if you grep for something which doesn't exist the exist code of the entire pipeline is 1, b/c grep didn't find it.